docs/src/guides/gatsby/deploy/_index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "How to Deploy Gatsby on Platform.sh"
+sidebarTitle: "Get started"
+weight: -110
+layout: single
+toc: false
+description: |
+    Create a Platform.sh account, download a few tools, and prepare to deploy Gatsby.
+---
+
+Gatsby is an open source framework based on React, that specializes in generating static websites from a variety of content sources. 
```

Comment:
I think this can be repeated as long as that upper navigation level exists. And, link to Gatsby.